program-runtime/src/compute_budget.rs

```diff
@@ -171,7 +171,7 @@ impl ComputeBudget {
             curve25519_ristretto_multiply_cost: 2_208,
             curve25519_ristretto_msm_base_cost: 2303,
             curve25519_ristretto_msm_incremental_cost: 788,
-            heap_size: solana_sdk::entrypoint::HEAP_LENGTH,
+            heap_size: u32::try_from(solana_sdk::entrypoint::HEAP_LENGTH).unwrap(),
```

Comment:
this `unwrap` should be safe since we are converting from a const, otherwise it _should_ panic.